### PR TITLE
Adding DICE to loss functions

### PR DESCRIPTION
**Pull request by kmader**
adding dice / dice_loss to the loss functions

### CODE REVIEW

File: keras_contrib/losses/__init__.py [hunk 1]
```diff
@@ -1 +1,22 @@
 from .dssim import DSSIMObjective
+import keras.backend as K
+
+def dice(y_true, y_pred):
+    """
+    A simple DICE implementation without any weighting
```
Comment (tboquet) [inline keras_contrib/losses/__init__.py]: Please use Keras style docstring.

File: keras_contrib/losses/__init__.py [hunk 2]
```diff
@@ -1 +1,22 @@
...
+    :param y_true:
+    :param y_pred:
+    :return:
+    """
+    y_t = K.batch_flatten(y_true)
+    y_p = K.batch_flatten(y_pred)
+    return 2.0*K.sum(y_t*y_p) / (K.sum(y_t)+K.sum(y_p)+K.epsilon())
```
Comment (tboquet) [inline keras_contrib/losses/__init__.py]: Please use pep8 compliant formatting.

**Comment by kmader** [comment]
was hoping to make the changes with the github editor rather than forking everything

**Comment by artvandelay** [comment]
I'm would like to work on this. Does OP mind? @kmader
